packages/platform-ios/native_modules.rb

```diff
@@ -5,24 +5,22 @@
 #
 require 'pathname'
 
-def use_native_modules!(root = "..", config = nil)
```

Comment:
Not needed, because `react-native config` is now `cwd` agnostic and will automatically resolve from the correct directory (project root).